### PR TITLE
chore: move color picker component to fast-tooling

### PR DESCRIPTION
**Pull request by williamw2**
# Pull Request

This PR moves the color picker component files from fast-tooling-react to fast-tooling and adjusts the import references in fast-tooling-react. 

### General

- [x] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

### CODE REVIEW

File: packages/tooling/fast-tooling/src/index.ts [hunk 1]
```diff
@@ -2,3 +2,4 @@ export * from "./schemas";
 export * from "./data-utilities";
 export * from "./message-system";
 export * from "./message-system-service";
+export * from "./web-components";
```
Comment (janechu) [inline packages/tooling/fast-tooling/src/index.ts:5]: Let's remove this as an export as the API is still in flux.

Comment (williamw2) [reply packages/tooling/fast-tooling/src/index.ts]: Done

File: packages/tooling/fast-tooling-react/src/form/custom-controls/control.css.utilities.tsx [hunk 2]
```diff
@@ -13,7 +13,7 @@ import {
     FASTSelect,
     FASTTextField,
 } from "@microsoft/fast-components";
-import { FASTColorPicker } from "./css";
+import { FASTColorPicker } from "@microsoft/fast-tooling//dist/esm/web-components";
```
Comment (radium-v) [inline packages/tooling/fast-tooling-react/src/form/custom-controls/control.css.utilities.tsx:16]: double-slash in the path

Comment (williamw2) [reply packages/tooling/fast-tooling-react/src/form/custom-controls/control.css.utilities.tsx]: Thank you. Fixed.

**Review by EisenbergEffect — APPROVED** [review]

**Review by janechu — APPROVED** [review]

**Pull request merged**